.github/workflows/ci.yml

```diff
@@ -399,7 +399,6 @@ jobs:
           ./babylonnode docker dependencies
       - name: "Execute User Flow: Install Core, Gateway and Monitoring on the same host"
         run: |
-          chmod +x  ./node-runner-cli/tests/userflows/install-docker-all-same-host.sh
```

Comment:
lets not remove that please. Someone will manage to change the permissions of that file in git and then we have to readd this. Happened to often in previous jobs, that I would like to avoid it. 
Even though here there is noone developing on Windows.